### DISCUSSION TITLE
Simple Process To Build Documentation Format

### DISCUSSION THREAD

**Discussion by ChrisLeeThompson** [gethinode/hinode#398] [Help] [answered]
I have attempted to configure the default hinode site using the create new site information shown in the getting started section of the Hinode documentation. I am wanting to build a Hinode site that looks like the documentation site for Hinode, but I am finding this is not a simple task.

I also tried to clone the Hinode docs page, but I encountered some errors when I ran hugo server.

Is there a simple example or way to convert the default Hinode site to a documentation style site - similar to the Hinode docs pages? I think the docs are very informative and useful, but on this one topic I am finding it difficult.

Thank you for any information you may provide!

**Comment by markdumay** [comment, marked as answer]
Thanks for your interest in Hinode. I can imagine it's difficult to start from scratch when creating a new docs site. I'm considering to write a new guide (and will add it to the backlog).

Meanwhile, the relevant part in the current docs can be found here: https://gethinode.com/docs/configuration/navigation/#sidebar-navigation. Is there a particular part you're struggling with?

The docs site requires npm. You can run the following commands to run it locally:

```bash
git clone https://github.com/gethinode/docs.git && cd docs
npm i && npm run mod:install
npm run start
```

I'm currently streamlining the repository to simplify the initial setup (with and w/o npm, see #392). It will be part of the upcoming `v0.18` release.

**Reply by ChrisLeeThompson** [reply]
Thank you for your reply and the information you provided. My limited experience with Hugo is perhaps making the learning curve a little steeper than usual :) The information about the side-bar navigation is helping. I think I was having trouble with adding content with the layout: docs header, but without adding the appropriate docs.yml file to the data directory. I will continue to experiment with adjusting the main template/theme to be more of a documentation site. Meanwhile, I will try using npm for running the documentation page locally.

Thank you, again, for the helpful information and a great Hugo theme!
